util/codec/bytes.go

```diff
@@ -78,7 +81,8 @@ func decodeBytes(b []byte, escapeFirst byte, escape byte, term byte) ([]byte, []
 		return nil, nil, errors.Errorf("insufficient bytes to decode value")
 	}
 
-	var r []byte
+	// Allocate more space to avoid unnecessary slice growing
+	r := make([]byte, 0, len(b)+20)
```

Comment:
the b may be large (e.g, containing multi different encoded slices), so allocing with `len(b) + 20` may be not proper here.